src/sections/Community/Handbook/repository.js

```diff
@@ -204,6 +206,58 @@ const Repository = () => {
                 </div>
               );
             })}
+            <a id="Maintainers">
+              <h2>Maintainers</h2>
+            </a>
+
+            {Maintainers.map((Maintainers, index) => {
+              const { category } = Maintainers;
+              return (
+                <div className="table-container" key={index}>
+                  <table className="frontendTable" key={category}>
+                    <thead>
+                      <tr>
+                        <th className="linkscol">Site</th>
+                        <th>Project</th>
+                        <th>Maintainers</th>
+                        <th className="linkscol">Repo</th>
+                      </tr>
+                    </thead>
+                    {Maintainers.subdata.map((subdata) => {
+                      const {
+                        project,
+                        maintainers,
+                        repository,
+                        site,
+                        image,
+                      } = subdata;
+                      return (
+                        <tbody key={project}>
+                          <tr>
+                            <td>
+                              <a href={site} target="_blank" rel="noreferrer">
+                                <img className="site-icon" src={image} />
+                              </a>
+                            </td>
+                            <td>{project}</td>
+                            <td>{maintainers}</td>
```

Comment:
@legendarykamal will you please ensure that each of these names are hyperlinked to their respective community member profiles on https://layer5.io/community/members?

Reply:
yes sure @leecalcote that would be great

Reply:
done